tools/ldap-server-ssl-check.cpp

```diff
@@ -10,11 +10,19 @@ int main(int argc, char** argv)
 	std::string uri(argv[1]);
 	setenv("LDAPTLS_REQCERT", "hard", 1);
 	setenv("LDAPTLS_CACERT", argv[2], 1);
-	try 
-	{
-		LDAPConnection lc( uri );
-		lc.start_tls();
-	}
+        try
+        {
+                LDAPConnection lc( uri );
+         try {
+             lc.bind();
+             exit(0);
+         } catch (LDAPException e) {
+                if (e.getResultCode() == 49) {
```

Comment:
42, 37, 49, 47 are meaningless numbers. `INVALID_CREDENTIALS` is what makes code understandable.
```
/usr/include/LDAPResult.h:        static const int INVALID_CREDENTIALS            = 49;
```